MediaGallerySynchronization/Model/SynchronizeFiles.php

```diff
@@ -85,7 +85,7 @@ public function execute(array $files): void
         foreach ($files as $file) {
             $path = $this->getFilePath($file);
             $time = $this->getFileModificationTime($file);
-            if (isset($assets[$path]) && $time === $assets[$path]) {
```

Comment:
Why you removing second part ?

Reply:
Basically to fix this issue we need to write new modified time to the metadata when we editing the file.

Reply:
` if (isset($assets[$path]) && $time === $assets[$path]) {`
This part is responsible to synchronize files that were modifiet since last execution time, so when we editing file we must change the modifiet date in metadata as well.